enterprise/cloud.amazon/src/org/netbeans/modules/cloud/amazon/serverplugin/AmazonJ2eePlatformImpl2.java

```diff
@@ -144,7 +144,7 @@ public boolean isToolSupported(String toolName) {
     }
 
     @Override
-    public Set getSupportedJavaPlatformVersions() {
+    public Set<String> getSupportedJavaPlatformVersions() {
```

Comment:
The method signature of the superclass hints this is correct, yet it is not generic by itself.